test/Market.test.ts

```diff
@@ -1028,28 +1026,34 @@ describe('Market', () => {
         userFixtures.buyer.bpBalance.sub(value)
       );
       expect(supplierFinalNoriBalance).to.equal(
-        supplierInitialNoriBalance.add(
-          purchaseAmount.sub(
-            formatTokenAmount(removalAmount)
-              .mul(project1HoldbackPercentage)
-              .div(100)
-              .add(
-                formatTokenAmount(removalAmount)
-                  .mul(project2HoldbackPercentage)
-                  .div(100)
-              )
+        supplierInitialNoriBalance
```

Comment:
Statements like these are prime candidates to check if there's an issue with our big number/floating point usage we discovered on a call with @rlfarman a week or so ago

Reply:
i'm not sure there's anything that interesting here because none of these values are floating point numbers, which was the source of that bug.  we have a ticket in this sprint to audit all of those instances so i'll look at them then.